lib/lighthouse/benefits_intake/service.rb

```diff
@@ -45,10 +45,18 @@ class InvalidDocumentError < StandardError; end
     # @param upload_url [String] override instance upload_url; optional, default = @location
     #
     def perform_upload(metadata:, document:, attachments: [], upload_url: nil)
```

Comment:
Thinking more about how we might roll this out - maybe teams could opt into the new perform_upload via feature flag? That way we wouldn't need all 4-5 teams involved to land the initial PR.